src/sonic-frr/patch/0048-bgpd-backpressure-fix-to-properly-remove-dest-for-bg.patch

```diff
@@ -46,23 +52,23 @@ index 2243ffdc77..b1f8f19594 100644
 +			zebra_announce_del(&bm->zebra_announce_head, dest);
 +		}
  	}
- 
  	bgp_evpn_remote_ip_hash_destroy(vpn);
+ 	bgp_evpn_vni_es_cleanup(vpn);
```

Comment:
Can you please clarify why is this change introduced?

Reply:
The patch was failing to be applied in its current shape. So, I had to manually apply the existing patch and recreate the patch file backwards. 
Note that bgp_evpn_vni_es_cleanup(vpn); is an existing code. It has not been added by me. This line has been autogenerated during patch creation by git diff command,

Reply:
Also, see in line number 34, we have a space. In the new patch apply, we have a strict whitespace check.

Reply:
Please do not add code that was not in the original patch.  You are asking for trouble.

Reply:
I haven't added this code. This is an auto generated code that is added by git format-patch as part of hunk formation.

Reply:
Makes sense. The git diff is little confusing since it doesn't differentiate between the + in the patch and + in git diff